api/db.py

```diff
@@ -0,0 +1 @@
+# Database access
```

Comment:
Would this be where we connect to the DB? (And in turn reference a secret .config file that we .gitignore?)

Reply:
Yes :) Thought I made it, but guess I didn't